devimint/src/bin/faucet.rs

```diff
@@ -116,7 +116,7 @@ async fn main() -> anyhow::Result<()> {
         .route(
             "/connect-string",
```

Comment:
TODO: rename faucet route to `/invite-code`?

Reply:
Rename it everywhere!